lib/govuk_schemas/random_example.rb

```diff
@@ -93,7 +93,7 @@ def payload(&block)
   private
 
     def customise_payload(payload)
-      original_payload = payload
+      original_payload = Marshal.load(Marshal.dump(payload)) # deep dup
```

Comment:
I wonder if it's worth adding a bit more detail to the commit message here about how this works, or wrapping this in a private method (`deep_dup`) to hide the internals, for the benefit of future readers, as I had to do a bit of Googling here to figure out what's going on under the hood!

Reply:
Yup, good suggestion. I've added a bit more of a comment and updated the commit message.